src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import logo from '../Header/OG-Spring.svg.png';
+import lists from '../../ExpandListItems.json';
+import './SideNav.scss';
+import NavElement from '../../components/NavElement/NavElement';
+import MobileNavElement from "../../components/MobileNavElement/MobileNavElement";
+
+const Navigation = () => {
+  const [showMobileNav, setShowMobileNav] = useState(false);
+
+  const handleClick = () => {
+    setShowMobileNav(!showMobileNav);
+  };
+
+  return (
+    <>
```

Comment:
No indentation. `Header` component is useless. You can use `div.header` instead of this fragment.

Reply:
This is still not resolved.